docs/workflow/building/coreclr/README.md

```diff
@@ -1,100 +1,158 @@
 
-### Build Drivers
+#### The Dev Shipping Packs
 
-If you want to use _Ninja_ to drive the native build instead of _Make_ on non-Windows platforms, you can pass the `-ninja` flag to the build script as follows:
+<!-- TODO: Link to the "using your build with the sdk" and "using your build with the shipping packages" docs, and rephrase accordingly, if needed. -->
+It is also possible to generate the full runtime NuGet packages and installer that you can use to test in a more production-esque scenario. To generate these shipping artifacts, you have to build the `clr`, `libs`, `host`, and `packs` subsets:
 
 ```bash
-./build.sh --subset clr --ninja
+./build.sh -subset clr+libs+host+packs -configuration Release
 ```
 
-If you want to use Visual Studio's _MSBuild_ to drive the native build on Windows, you can pass the `-msbuild` flag to the build script similarly to the `-ninja` flag.
+The shipping artifacts are placed in the `artifacts/packages/<Configuration>/Shipping` directory. Here, you will find several NuGet packages, as well as their respective symbols packages, generated from your build. More importantly, you will find a zipped archive with the full contents of the runtime, organized in the same layout as they are in the official dotnet installations. This archive includes the following files:
+
+- `host/fxr/<net-version>-dev/hostfxr` (`hostfxr` is named differently depending on the platform: `hostfxr.dll` on Windows, `libhostfxr.dylib` on macOS, and `libhostfxr.so` on Linux)
+- `shared/Microsoft.NETCore.App/<net-version>-dev/*` (The `*` here refers to all the libraries dll's, as well as all the binaries necessary for the runtime to function)
+- `dotnet (dotnet.exe on Windows)` (The main `dotnet` executable you usually use to run your apps)
+
+Note that this package only includes the runtime, therefore you will only be able to run apps but not build them. For that, you would need the full SDK.
+
+**NOTE:** On Windows, this will also include `.exe` and `.msi` installers, which you can use in case you want to test your build machine-wide. This is the closest you can get to an official build installation.
+
+For a full guide on using the shipping packages for testing checkout the doc we have dedicated to it [over here](/docs/workflow/testing/using-dev-shipping-packages.md).
 
-We recommend using _Ninja_ for building the project on Windows since it more efficiently uses the build machine's resources for the native runtime build in comparison to Visual Studio's _MSBuild_.
+### Cross Compilation
+
+Using an x64 machine, it is possible to generate builds for other architectures. Not all architectures are supported for cross-compilation however, and it's also dependant on the OS you are using to build and target. Refer to the table below for the compatibility matrix.
+
+| Operating System | To x86   | To Arm32 | To Arm64 |
+| :--------------: | :------: | :------: | :------: |
+| Windows          | &#x2714; | &#x2714; | &#x2714; |
```

Comment:
The Windows->Arm32 support is only for diagnostic tooling (which actually runs on x86) ever since Windows ARM was removed. I think we should remove that check mark.

```suggestion
| Windows          | &#x2714; |          | &#x2714; |
```

Reply:
I see, so as of now we only support ARM32 per se on Linux? Or is Linux support also dropped? In this case, then we could remove the ARM32 column.

Reply:
Linux still supports building the runtime targeting ARM32, so we should keep the column.